src/Atc.Network/Helpers/KnowTcpUdpPortsLookupHelper.cs

```diff
@@ -0,0 +1,110 @@
+// ReSharper disable SwitchStatementHandlesSomeKnownEnumValuesWithDefault
+// ReSharper disable UnusedMember.Local
+namespace Atc.Network.Helpers;
+
+/// <summary>
+/// KnowIPPortsLookupHelper.
+/// </summary>
+/// <remarks>
+/// https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers
+/// </remarks>
+public static class KnowTcpUdpPortsLookupHelper
+{
+    public static bool IsKnow(
+        ServiceProtocolType serviceProtocolType,
+        int portNumber)
+    {
+        return serviceProtocolType switch
+        {
+            ServiceProtocolType.Https => Https.Contains(portNumber),
+            ServiceProtocolType.Http => Http.Contains(portNumber),
+            ServiceProtocolType.Ftps => Ftps.Contains(portNumber),
+            ServiceProtocolType.Ftp => Ftp.Contains(portNumber),
+            ServiceProtocolType.Telnet => Telnet.Contains(portNumber),
+            ServiceProtocolType.Ssh => Ssh.Contains(portNumber),
+            _ => false,
+        };
+    }
+
+    private static readonly List<int> Https = new()
```

Comment:
The maximum port number is 65535 which is 16 bits and can be accurately represented as an ushort.